docs/docs/references/applications/README.mdx

```diff
@@ -0,0 +1,67 @@
+import ApplicationTypesIntro from '../../fragments/_application_types_intro.mdx';
+
+# 📱 Applications
+
+## Introduction
+
+Each real-world application that requires user authentication using Logto needs to be registered as an application in Logto.
+
+## Application ID
+
+_Application ID_ is a unique auto-generated identifier to identify your application with the Logto application.
+
+## Application Types
+
+We provide three application types to help developers integrate their applications with Logto:
```

Comment:
nit: Instead of `provide`, more like we categorize the application into three types? I'm not sure

Reply:
We will provide more application types in the future. So I think it's ok to use provide.